staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/testing/testcase.go

```diff
@@ -76,7 +77,7 @@ func NewFakeDataSource(name string, webhooks []registrationv1beta1.Webhook, muta
 	return client, informerFactory
 }
 
-func newAttributesRecord(object metav1.Object, oldObject metav1.Object, kind schema.GroupVersionKind, namespace string, name string, resource string, labels map[string]string) admission.Attributes {
+func newAttributesRecord(object metav1.Object, oldObject metav1.Object, kind schema.GroupVersionKind, namespace string, name string, resource string, labels map[string]string, annotations map[string]string) admission.Attributes {
```

Comment:
Instead of adding yet another argument to this, prefer exposing `FakeAttributes` and `FakeAttributes.Annotations` (or if you prefer, `FakeAttributes.GetAnnotations`)

Reply:
Done.